app/templates/components/supplementary-file-browser.hbs

```diff
@@ -0,0 +1,26 @@
+{{#file-renderer download=selectedFile.links.download
+    width="100%" height="700"}}
+{{/file-renderer}}
+
+{{#if supplementList}}
+    <h5>{{selectedFile.name}}</h5>
+    <section>
+        <button class="show-more-button text-muted left" {{action 'moveLeft'}}>
+            <i class="fa fa-caret-left"></i>
+        </button>
+        {{#liquid-bind supplementList elementId="file-list" tagName="ul" class="p-l-md p-r-md" use=scrollAnim as |list|}}
+            {{#each list as |supplement|}}
+                <li class="small-files col-xs-2 p-md box {{if (eq selectedFile supplement) 'selected'}}">
+                    {{#if (eq supplement.kind "folder")}}
+                        <i class="fa fa-folder fa-3x" aria-hidden="true" href={{concat model.baseUrl model.project}}><h6>{{supplement.name}}</h6></i> {{!Actually make a link here}}
+                    {{else}}
+                        <i class="fa fa-file-text fa-3x" aria-hidden="true" {{action 'changeFile' supplement}}><h6>{{supplement.name}}</h6></i>
```

Comment:
For accessibility reasons, we try to avoid markup that treats text elements (`span`, `i`, etc) as clickable. Screen readers don't automatically know to apply "click here" behaviors, because they use markup as a guide to what the document does.

For example, we might rewrite this as `<button {{action 'lightsCamera'}}><i class"fa fa-lorna">{{supplement.name}}</i>`

Is the heading tag a requirement for some other library? Screen readers also use headings to indicate a document headline, so using them for label text may be  confusing.